modules/textures/package.json

```diff
@@ -42,6 +42,7 @@
     "build-crunch-worker": "webpack --entry ./src/workers/crunch-worker.ts --output ./dist/crunch-worker.js --config ../../scripts/webpack/worker.js"
   },
   "dependencies": {
+    "@loaders.gl/core": "3.0.1",
```

Comment:
We need to break this dependency. Maybe create an issue for that.

Reply:
Done